### PR TITLE
Create the list structure of todolist app

### PR DESCRIPTION
**Pull request by NitBravoA92**
# 🎏 In this Milestone, I:

✅ Installed and set up webpack, webpack dev server, html webpack plugin and a CSS Loader. 
✅ Created an index.html file in the src folder, with a main section to display the tasks list
✅ Created a style.css file in the src folder with all the CSS styles for the main section of the To-Do List
✅ Created an index.js file in the src folder
✅ Created an array of objects in index.js file, with all the information about each task in To Do list
✅ Created a function to generate the markup of each task
✅ Created a function to iterate over the tasks array and populate an HTML list item element for each task
✅ Did render each task item dynamically inside the To Do list container when the page loads
✅ Did render each task item dynamically inside the To Do list container when the page loads
✅ I generated the final application files in the ./dist folder using webpack

# ⭐To the code reviewer 👨‍💻
💝 I really thank 🙇‍♂️ you, dear code reviewer 👨‍🎨 for dedicating your precious 🥇 time ⌚ to check the PR 🧾 .

I kindly ask 🙏 dear code reviewer 🤵 that if there is any kind of issue 🦯 in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions 🎁 if needed.

If you think there are some big issues that are essential to be changed ♻️ please kindly contact 📞 me through Zoom or Slack, I'am available on Microverse UTC-6 time zone.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+#app #title,
+#app #to-do-list .task {
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #d7d7d7;
+}
```
Comment (zieeco) [inline src/style.css:49]: - Hey, great job with your design, however, there is to much space here, kindly update your styling. see the demo video to fix the issue below to improve the user experience of your app.

image

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <main>
+    <section id="app">
+      <h2 id="title"><span>Today's To Do</span> 
+        <button type="button" id="refresh">
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="#bebebe" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M21.5 2v6h-6M2.5 22v-6h6M2 11.5a10 10 0 0 1 18.8-4.3M22 12.5a10 10 0 0 1-18.8 4.2"/></svg>
+        </button>
+      </h2>
+      <ul id="to-do-list"></ul>
```
Comment (zieeco) [inline src/index.html:17]: - Hey, from the image I added in the previous review, your design is missing the `clear completed` task button. The button is meant to be in the app so users can clear the list of tasks they have completed which is the subsequent project milestone, you will implement this functionality. please, kindly add the missing button.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @NitBravoA92,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- ToDo List structure setup correctly 🔥 
- Good commit message. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the co

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @NitBravoA92, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added clear all completed button as suggested ✔️ 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
- Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**